### PR TITLE
Rename ejabberd.cfg to mongooseim.cfg

### PR DESCRIPTION
**Pull request by kzemek**

### CODE REVIEW

**Comment by mongoose-im** [comment]
[5555.1](https://travis-ci.org/esl/MongooseIM/jobs/425753571) / Erlang 19.3 / small_tests / dd60fd27f37bcd8d7619c3115f03b29d8599233b
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2071/5555/small_tests.19.3)

**Comment by mongoose-im** [comment]
[5565.1](https://travis-ci.org/esl/MongooseIM/jobs/427115811) / Erlang 19.3 / small_tests / c1104faddb32dbc9f6b6a48d0a76225087786f09
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2071/5565/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2071/5565/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-09-11_09.28.33/lib.mongooseim.logs/run.2018-09-11_09.28.34/suite.log.html)

**Comment by michalwski** [comment]
@kzemek @fenek I just noticed in `ejabberd_config.erl` lines like this:

```
Problem loading ejabberd config file
```

Do we want to rename it 

```
Problem loading MongooseIM config file
```

in this PR?

**Comment by fenek** [comment]
@michalwski 

I think that replacing "ejabberd" leftovers may be a subject of separate PR. They may be found in many places... :)

**Comment by mongoose-im** [comment]
[5586.1](https://travis-ci.org/esl/MongooseIM/jobs/429498183) / Erlang 19.3 / small_tests / 327750c009903b1465810d3afe75d4fa3186be10
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2071/5586/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2071/5586/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-09-17_10.34.31/lib.mongooseim.logs/run.2018-09-17_10.34.31/suite.log.html)

File: doc/user-guide/ICE_tutorial.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ In order to enable signalling we need an instance of [MongooseIM] running with t
 #### Configuration
 
 You can find MongooseIM installation instructions on [this page](Getting-started.md).
-Once you have cloned the repository and compiled the project, you need to modify the `ejabberd.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/ejabberd.cfg`, where `$REPO` is a top-level directory of
+Once you have cloned the repository and compiled the project, you need to modify the `mongooseim.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/mongooseim.cfg`, where `$REPO` is a top-level directory of
```
Comment (goddammit) [inline doc/user-guide/ICE_tutorial.md]: Once you have cloned the repository and compiled the project, you need to modify the `mongooseim.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/mongooseim.cfg`, where `$REPO` is a top-level directory of the cloned repo).

File: doc/user-guide/Jingle-SIP-setup.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Now we need to use this to update `/etc/hosts` file like below:
 
 At this point I assume that MongooseIM was built with `make rel`, that it is running and the current working directory is `_build/prod/rel/mongooseim`.
 Similar to Routr, MongooseIM also needs to know which hosts to server.
-Please replace the default host defined in file `etc/ejabberd.cfg` in line
+Please replace the default host defined in file `etc/mongooseim.cfg` in line
```
Comment (goddammit) [inline doc/user-guide/Jingle-SIP-setup.md]: Please replace the default host defined in `etc/mongooseim.cfg` in line ?

File: src/mod_carboncopy.erl [hunk 3]
```diff
@@ -3,7 +3,7 @@
 %%% Author  : Eric Cestari <ecestari@process-one.net>
 %%% Purpose : Message Carbons XEP-0280 0.8
 %%% Created : 5 May 2008 by Mickael Remond <mremond@process-one.net>
-%%% Usage   : Add the following line in modules section of ejabberd.cfg:
+%%% Usage   : Add the following line in modules section of mongooseim.cfg:
```
Comment (goddammit) [inline src/mod_carboncopy.erl]: Add the following line in the modules section of mongooseim.cfg:

**Comment by mongoose-im** [comment]
[5589.1](https://travis-ci.org/esl/MongooseIM/jobs/429529108) / Erlang 19.3 / small_tests / 80cbea4dea381cf2f5b9573d82dbeeeca3eef171
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2071/5589/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2071/5589/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-09-17_13.06.52/lib.mongooseim.logs/run.2018-09-17_13.06.53/suite.log.html)

**Pull request merged**